### PR TITLE
≈Michael's first draft of take home assignment

### PR DESCRIPTION
**Pull request by rmikelev**
Finished nav bar with active links
two columns with real text using white background
footer with site map

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,97 @@
...
+                </li>
+              </ul>
+              </div>
+          </div>
+        </div>
+        <div class="row align-items-center">
+          <div class=" sect1 col px-3 "> 
+       
+            <h4 class="text-center p-1">Cloak of Fear©</h4>
+              <h6 class="text-sm-center .rem">(6-17-2004)</h6>
```
Comment (ayojackay) [inline index.html:41]: I see a class called `.rem` is used here but I don't see it listed in your styles.

**Review by ayojackay — APPROVED** [review]
Looks great! Awesome job, Michael! I just had a comment about something but other than that, all good. Thank you very much!

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/* .container {
+    background-color: #FFE5A1;
+} */
+
+.nav justify-content-center {
```
Comment (ayojackay) [inline css/styles.css:5]: hmmm there isnt a tag called `justify-content-center`, did you mean call a class of this? like so: `.justify-content-center`

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    align-items: center;
```
Comment (ayojackay) [inline css/styles.css:6]: without activating `flex`, the style property `align-items` will not apply. Make sure to add `display: flex`

**Comment by rmikelev** [comment]
Jacque,

Thank you! .rem was an error, thank you for spotting it, that I forgot to remove. Done!  Also .nav justify-content-center was unnecessary  since that was done in bootstrap so I just removed that also. Not quite there in choosing what is to be in css and what in bootstrap. Thank you for this excellent learning experience. I appreciate that you took me on as a collaborator. Will see you tonight in class.

Michael



Michael Levine
***@***.***
559-250-4137
877 East Woodhaven Lane
Fresno, CA 93720

**Comment by ayojackay** [comment]
Thanks, Mike, I saw you made the changes. I went ahead and resolved. This is still approved! Thank you!

**Pull request merged**